akka-actor-typed/src/main/scala/akka/actor/typed/internal/MiscMessageSerializer.scala

```diff
@@ -1,36 +1,36 @@
 /**
  * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster.typed.internal
+package akka.actor.typed.internal
 
+import java.io.NotSerializableException
 import java.nio.charset.StandardCharsets
 
+import akka.actor.typed.{ ActorRef, ActorRefResolver }
+import akka.actor.typed.scaladsl.adapter._
 import akka.annotation.InternalApi
 import akka.serialization.{ BaseSerializer, SerializerWithStringManifest }
-import akka.actor.typed.ActorRef
-import akka.cluster.typed.ActorRefResolver
-import akka.actor.typed.scaladsl.adapter._
-import java.io.NotSerializableException
 
 @InternalApi
 class MiscMessageSerializer(val system: akka.actor.ExtendedActorSystem) extends SerializerWithStringManifest with BaseSerializer {
 
   private val resolver = ActorRefResolver(system.toTyped)
+  private val actorRefManifest = "a"
```

Comment:
I think in the other serializers we use the constant-style naming `ActorRefManifest` so we don't have to use back ticks in the pattern matching (not that it matters much).